package.json

```diff
@@ -4,9 +4,8 @@
   "private": true,
   "dependencies": {
     "@azure/storage-blob": "^12.14.0",
-    "@fuel-wallet/sdk": "^0.16.1",
```

Comment:
Didn't find it used anywhere so removed it. The package has been deprecated for a while now already and `fuels` should be used in its place.